### ISSUE TITLE
Strange look on content

### ISSUE THREAD

**Issue by andwah** [StarcounterSamples/AppStoreClient#27]
Using latest 3.1.3

![image](https://cloud.githubusercontent.com/assets/2656996/11362344/3d00a830-9293-11e5-91cf-3f4dd029433e.png)

**Comment by andwah** [comment]
In the brower console window:

![image](https://cloud.githubusercontent.com/assets/2656996/11362603/18634788-9295-11e5-85b2-2faee239a0b2.png)

**Comment by warpech** [comment]
First I need to build it, because the current version of App Store Client is not in the app store :( (last published version is 3.0.2)

**Comment by warpech** [comment]
Actually seems that last 5 versions were not published in the app store! Seems we have some release loophole

**Comment by warpech** [comment]
For me it also doesn't work but with different result: 

![screen shot 2015-11-24 at 10 38 17](https://cloud.githubusercontent.com/assets/566463/11363038/9a5ec260-9297-11e5-9296-b273fd117246.png)

**Comment by warpech** [comment]
Seems that I had a timeout on a call to port 8182... It is blocked by Windows Firewall by default.

Now I managed to reproduce the problem.

**Comment by warpech** [comment]
Regarding Windows Firewall... Seems that whoever sets the "Allow scnetworkgateway*" needs to set up another rule for port 8182. @andwah should this be another issue?

![image](https://cloud.githubusercontent.com/assets/566463/11363260/056a6536-9299-11e5-8252-d8cdb3867152.png)

**Comment by andwah** [comment]
Port 8182 is "hardcode" so it needs to be set up from incoming requests (from the appstore client) on the starcounter server.

**Comment by warpech** [comment]
Yeah but all other ports (see above screenshot) are added automatically by Starcounter to Windows Firewall. I didn't add them manually. So why should I add 8182 manually?

**Comment by warpech** [comment]
The problem in topic was caused by `owl.carousel/.gitignore` ignoring the files in the dist folder. I fixed it by changing the `.gitignore` file and pushing files from the 3.1.3 zip package. 

See commit https://github.com/StarcounterSamples/AppStoreClient/commit/2f3b7bf47117fc18d425ffef6286149e51e5142f for details.

I released the fix as 3.1.4.

Thanks @andwah for investigation!

**Comment by andwah** [comment]
> So why should I add 8182 manually?

I agree that the port 8182 should also be managed in the same way the other ports do, Automatically.

I created a separate issue for it. #28

**Comment by warpech** [comment]
Nice! Thanks!
